holmes.go

```diff
@@ -170,6 +170,11 @@ func (h *Holmes) goroutineCheckAndDump(gNum int) {
 		return
 	}
 
+	if h.opts.GrOpts.GoroutineTriggerNumMax != 0 && gNum >= h.opts.GrOpts.GoroutineTriggerNumMax {
```

Comment:
move to `matchRule`?

Reply:
oh~  cpu and mem not have the parameter.

Reply:
`matchRule` tool function, no method, I couldn't log inside unless I pass a holmes class variable or I change it to a method of holmes class.

Reply:
`matchRule` is a tool func that should support all types dump.
Is it possible for us to add a parameter of max, the default value for cpu and mem is 0 ? 
`func matchRule(history ring, curVal, ruleMin, ruleAbs, ruleDiff int, ruleMax) bool`
what's your opinion?

Reply:
OK, and we dont need log? just return a false

Reply:
https://github.com/mosn/holmes/blob/04a2f1660335c5ef1178718f009fe38c6b6c78ea/holmes.go#L187
this log is ok

Reply:
yes, I can add a max parameter for debugUniform.

Reply:
For this change, and I think that It would be easy if mem also need to support max in some scenarios later.

Reply:
got it!